test/exactly-once/exactlyonceck/exactlyonceck

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test that a catalog created by materialized v0.1.0 is compatible with the
+# version of materialized under test.
+
+set -euo pipefail
+
+say() {
+    echo "### $* ###"
+}
+
+run_sql() {
+    psql -U materialize -h localhost -p 6875 materialize -c "\pset footer off" -c "$1"
+}
+
+launch_materialized() {
+    name=$1
+    shift
+    "materialized-$name" -w1 "$@" &
+    materialized_pid=$?
+    wait-for-it --timeout=30 -q localhost:6875
+    run_sql "SELECT 1" > /dev/null
+}
+
+kill_materialized() {
+  kill "$materialized_pid"
```

Comment:
I think ```SIGKILL``` needs to be used here. Otherwise, over the long term I think we will eventually be intercepting ```SIGTERM``` so this function will actually begin to do ```shutdown_materialized_cleanly``` which is probably not the intent.